src/libhirte/common/cfg.h

```diff
@@ -175,16 +175,4 @@ bool cfg_get_bool_value(struct config *config, const char *option_name);
  */
 bool cfg_s_get_bool_value(struct config *config, const char *section, const char *option_name);
 
-/*
- * Iterate over configuration and for each item execute the iter function.
- * For example using following iterator method would print the whole configuration:
- *
- *     bool config_option_iterator(const void *item, void *udata) {
- *         const struct config_option *option = item;
- *         printf("%s = %s\n", option->name, option->value);
- *         return true;
- *     }
- *
- * WARNING: configuration items are accessed directly, DO NOT modify them during iteration!!!
- */
-void cfg_iterate(struct config *config, bool (*iter)(const void *item, void *udata), void *udata);
+const char *cfg_dump(struct config *config);
```

Comment:
Could you please add a comment about the function similar to the comments we have for other functions here?

Reply:
Yes, will do :)